Feature branch

aracelicaes opened the pull request:
Pull request to check the linters.

HeyItsGwen committed:
make skeleton, add files, add basic reset

add basic file structure, skeleton to html, scss file and their skeleton, and a basic reset

HeyItsGwen committed:
add linter files

add and configure linting files (stickler, stylelint)

HeyItsGwen committed:
add grid system section to scss

HeyItsGwen committed:
start basic framework scss

make framework file, make a flex class, and a row class to contain the columns.

aracelicaes committed:
Created small medium and large cols

Created small medium and large columns for their pertaining media queries. Created mixins for the widths and the padding. Compiled code.

HeyItsGwen committed:
start building blocks for odin website

make the chunky colorful building blocks for the nav, top info, images, and how it works for the odin project website

aracelicaes committed:
Added 3 sections on HTML and CSS

Added 3 sections with general content on HTML added stylings on CSS. Created a mixin with the max-width margin and width of the inner containers. Added colsm-12 and col-lg-6 to HTML for the Success Stories Section.

aracelicaes committed:
Added to Framework

Created hidden and show maxins added to framework the classes to hide and show the content accordingly.

aracelicaes committed:
Added Footer in HTML

Created Footer and added HTML content.

HeyItsGwen committed:
add images

add images from site to images folder

HeyItsGwen committed:
make footer work

add font awesome link to html, add font awesome icons to footer, make styling of icons and footerLinks work like the reference website.

HeyItsGwen committed:
add images to success stories

add images to success stories using background-image and positioning, and style to container to look like the reference

HeyItsGwen committed:
style success stories and start learning section

add styles and positioning to elements in success stories and the start learning sections to make it look like the reference.

aracelicaes committed:
Added scss stylings to two sections

Added background-color: margin top margin bottom, button characteristics, backgrdound color hover color shadow and padding.

aracelicaes committed:
Inserted HTML and CSS for insideTop

Added color, background-gradient, button color, H1 font size weight and color. Same for paragraph.

HeyItsGwen committed:
finish nav bar

make nav bar responsiveness work, and add styling to logo, link list, and burger icon. make hover stying work.

aracelicaes committed:
delted comments

deleted unnecessary comments.

aracelicaes committed:
deleted incorrect align-items prop

delted it because it wanst affecting code and it doesnt take space around .

HeyItsGwen committed:
remove stickler

it's bad

HeyItsGwen committed:
Merge branch 'Feature-Branch' of https://github.com/HeyItsGwen/Grid-Based-Framework-Creation into Feature-Branch

HeyItsGwen committed:
more stickler files

yay

HeyItsGwen committed:
stickler yay

HeyItsGwen committed:
retry stickler

HeyItsGwen committed:
Update .stickler.yml

HeyItsGwen committed:
Create .stylelintrc.json

stickler-ci committed:
Fixing style errors.

stickler-ci committed:
Fixing style errors.

HeyItsGwen committed:
making stickler work

yeah, it's really bad

HeyItsGwen committed:
Merge branch 'Feature-Branch' of https://github.com/HeyItsGwen/Grid-Based-Framework-Creation into Feature-Branch

HeyItsGwen committed:
more stickler ignores

trying to ignore scss files

HeyItsGwen committed:
fix stickler things

Woo

HeyItsGwen committed:
even more stickler

HeyItsGwen committed:
add instructions to readme

HeyItsGwen committed:
Update README.md

HeyItsGwen committed:
Update README.md

HeyItsGwen committed:
Update README.md

HeyItsGwen committed:
Update README.md

aracelicaes committed:
Edited Read Me file

Added About section, technologies and Acknowledgements.

shshamim63 commented:
## Status :red_circle: [Changes Required] :red_circle: 
___
Hi @aracelicaes  and @HeyItsGwen :wave: ,
Great work :clap: . Most of the items are here to decorate the page. But some items are causing overflow of this page. You are trying to clone a webpage by using your own generated framework. But if it causing an overflow, that does not seem right. Isn't it? :thinking: .
#### Please follow the instruction below to complete this project.
___
## :spades:  [Mandatory Changes] :spades: 
___
### :desktop_computer: Large View
1. I think the row declaration with margin: auto 2% is causing it. Instead of it you can use margin: auto auto.
![task1](https://user-images.githubusercontent.com/38625956/69459255-9179ec00-0d9b-11ea-973d-210a42db68dd.png)
___
### :computer: Medium View
1. The same as above problem. Item is overflowing. If you inspect the element you will get the cause. Try to minimize the window, and you will notice the cause.
![task2](https://user-images.githubusercontent.com/38625956/69459476-064d2600-0d9c-11ea-8952-c45cb098c9d4.png)
___
## :clubs: [Optional Changes] :clubs: 
___
Well this project  main task is to build your own customized framework. So the number of features you will add, it will become more functional and user friendly. I will suggest the following aspects :point_down: 
#### 1. Margins
⋅⋅* The CSS files define separate classes for margin top, bottom, left, right, all margins and auto
⋅⋅* The margins should span from 0 rem to 1 rem  
### 2. Text
..* At least five classes to set the font size of an element
..* Three classes to set the font weight of an element (bold, normal, medium)
### 3. Border
1. The CSS files define separate classes for border top, bottom, left, right, all border
2. The border should span from 0 to 1rem
3. The border should have classes for making box round
4. The border should have a class defined to transform a box into a circle

___
### Please fix the mentioned issue and send another review request.
## :herb:  Happy Coding ;herb
### <em> Shakhawat Hossain </em>
[Linkedin](https://www.linkedin.com/in/shakhawathossainshamim/) | [Slack](Shakhawat Hossain)

HeyItsGwen committed:
add overflow hidde

stickler-ci committed:
Fixing style errors.

stickler-ci committed:
Fixing style errors.

aracelicaes committed:
changed margins on How Works section

changed to margin auto to remove the overflow.

aracelicaes committed:
Merge branch 'Feature-Branch' of https://github.com/HeyItsGwen/Grid-Based-Framework-Creation into Feature-Branch

aracelicaes committed:
added change to the margin

added change to margin for the How Works section to center and remover overflow.

stickler-ci committed:
Fixing style errors.

stickler-ci committed:
Fixing style errors.

aracelicaes committed:
corrected linter

corrected linters

aracelicaes committed:
committing changes for stickler

stickler corrections.

aracelicaes committed:
applied linter corrections

corrected stickler errors

suretrust reviewed (approved):
### Amazing job guys, let's get this merged! :+1: 

_Good luck with your future projects and happy coding!_ :tada:
